src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -145,7 +145,6 @@
   <!-- Sources -->
   <ItemGroup>
       <Compile Include="resources\SR.extra.cs" />
```

Comment:
Move these extras to the common file?

There are going to be strings in the shared file that are used by one runtime and not the other one. We should figure out how to tech linker to deal with it, or something like that.